terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -30,6 +32,8 @@
 
     protected long defaultTimeoutSecForImplicitlyWait = 5;
 
+    private static final Logger logger = LoggerFactory.getLogger(WebDriverOperations.class);

```

Comment:
69300e665b02d08b2259c24d8639a5ef710d3c8b 
I made the order of static fields -> instance fields at above commit.